api/app/Http/Middleware/AuthMiddleware.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ * This file is part of the Spira framework.
+ *
+ * @link https://github.com/spira/spira
+ *
+ * For the full copyright and license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace App\Http\Middleware;
+
+use Closure;
+use Illuminate\Http\Request;
+use Spira\Contract\Exception\UnauthorizedException;
+
+class AuthMiddleware
```

Comment:
Please rename to AuthorizationMiddleware to differentiate from Authentication (two separate concepts)